index.html

```diff
@@ -18,16 +21,178 @@
             </div>
         </nav>
     </header>
-    <!-- END NAVBAR SECTION-->
+    <!-- End NAVBAR SECTION-->
 
-    <!-- START HEADLINE SECTION-->
+    <!-- Start HEADLINE SECTION-->
     <section class="headline">
         <div class="content">
             <h1 class="author">Thiri Htet Htet Aung</h1>
             <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
             <button type="button" class="collab-btn">Start collaboration</button>
         </div>
     </section>
-    <!-- END HEADLINE SECTION -->
+    <!-- End HEADLINE SECTION -->
+
+    <!-- Start WORK SECTION -->
+    <section class="works">
+        <!-- start title -->
+        <div class="title">
+            <h1>Projects</h1>
+        </div>
+        <!-- end title -->
+
+        <!-- start project card -->
+        <div class="projects">
+            <!-- start card1 -->
+            <div class="project-cards project1">
+                <div class="pjimgs">
+                    <img src="./img/pj1.svg" alt="pj1"/>
+                </div>
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
```

Comment:
As required in the project all occurrences of a button should have multiple states (hover, active, and so on) to make them visually appealing.
I suggest you style these states as given in the figma template, see below

<img width="589" alt="btn states" src="https://user-images.githubusercontent.com/84930355/205227361-7d272cd1-b223-4dd5-bd13-d2236ffbe216.png">

Reply:
The project btn style isn't included in one of those three style what should I do for those btns?  Should I just Leave like this?@DuaneDave 
![Screenshot 2022-12-02 205208](https://user-images.githubusercontent.com/83853383/205314231-d0681a12-67ef-4a5a-b4a7-923b003e3717.png)